sdk/appcenter/src/main/java/com/microsoft/appcenter/utils/storage/DatabaseManager.java

```diff
@@ -508,38 +379,28 @@ SQLiteDatabase getDatabase() throws RuntimeException {
     }
 
     /**
-     * Switches to in-memory management, triggers error listener.
+     * Sets {@link SQLiteOpenHelper} instance.
      *
-     * @param operation The operation that triggered the error.
-     * @param exception The exception that triggered the switch.
+     * @param helper A {@link SQLiteOpenHelper} instance to be used for accessing database.
      */
     @VisibleForTesting
-    void switchToInMemory(String operation, RuntimeException exception) {
-
-        /* Create an in-memory database. */
-        mIMDB = new LinkedHashMap<Long, ContentValues>() {
-
-            @Override
-            protected boolean removeEldestEntry(Entry<Long, ContentValues> eldest) {
-                return IN_MEMORY_MAX_SIZE < size();
-            }
-        };
-
-        /* Trigger error listener. */
-        if (mListener != null) {
-            mListener.onError(operation, exception);
-        }
+    void setSQLiteOpenHelper(@NonNull SQLiteOpenHelper helper) {
+        mSQLiteOpenHelper.close();
+        mSQLiteOpenHelper = helper;
     }
 
     /**
-     * Sets {@link SQLiteOpenHelper} instance.
+     * Gets an array of column names in the table.
      *
-     * @param helper A {@link SQLiteOpenHelper} instance to be used for accessing database.
+     * @return An array of column names.
      */
     @VisibleForTesting
-    void setSQLiteOpenHelper(@NonNull SQLiteOpenHelper helper) {
-        mSQLiteOpenHelper.close();
-        mSQLiteOpenHelper = helper;
+    String[] getColumnNames() {
+
+        // TODO: Below line doesn't look efficient to get column names. Could use "PRAGMA table_info(table-name)" to avoid getting all data back from database just to get column names.
```

Comment:
How about let's do this TODO now? 😄

Reply:
This is used only in unit tests we could just remove the comment